qemu/tests/qmp_command.py

```diff
@@ -200,41 +162,43 @@ def check_result(qmp_o, output=None, exception_list=""):
         qmp_port = qmp_ports[0]
     else:
         test.error("Incorrect configuration, no QMP monitor found.")
-    hmp_ports = vm.get_monitors_by_type('human')
-    if hmp_ports:
-        hmp_port = hmp_ports[0]
-    else:
-        test.error("Incorrect configuration, no QMP monitor found.")
     callback = {"host_cmd": lambda cmd: process.system_output(cmd, shell=True).decode(),
                 "guest_cmd": session.cmd_output,
-                "monitor_cmd": hmp_port.send_args_cmd,
                 "qmp_cmd": qmp_port.send_args_cmd}
 
     def send_cmd(cmd):
-        """ Helper to execute command on ssh/host/monitor """
+        """ Helper to execute command on host/ssh guest/qmp monitor """
         if cmd_type in callback.keys():
```

Comment:
Miss doc string for "cmd"

Reply:
It is too simple, i think it is redundant to explain `cmd` is a command to be execute again.